cmd/ebrelayer/relayer/relayer.go

```diff
@@ -0,0 +1,119 @@
+package relayer
+
+// -----------------------------------------------------
+//      Relayer
+//
+//      Initializes the relayer service, which parses,
+//      encodes, and packages named events on an Ethereum
+//      Smart Contract for validator's to sign and send
+//      to the Cosmos bridge.
+// -----------------------------------------------------
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	amino "github.com/tendermint/go-amino"
+
+	sdkContext "github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/swishlabsco/cosmos-ethereum-bridge/cmd/ebrelayer/contract"
+	"github.com/swishlabsco/cosmos-ethereum-bridge/cmd/ebrelayer/events"
+	"github.com/swishlabsco/cosmos-ethereum-bridge/cmd/ebrelayer/txs"
+)
+
+// -------------------------------------------------------------------------
+// Starts an event listener on a specific network, contract, and event
+// -------------------------------------------------------------------------
+
+func InitRelayer(cdc *amino.Codec, chainId string, provider string,
+	contractAddress common.Address, eventSig string,
+	validatorFrom string) error {
+
+	validatorAccAddress, validatorName, err := sdkContext.GetFromFields(validatorFrom)
+	if err != nil {
+		fmt.Printf("failed to get from fields: %v", err)
+		return err
+	}
+	validatorAddress := sdk.ValAddress(validatorAccAddress)
+
+	passphrase, err := keys.GetPassphrase(validatorFrom)
+	if err != nil {
+		return err
+	}
+
+	//Test passhprase is correct
+	_, err = authtxb.MakeSignature(nil, validatorName, passphrase, authtxb.StdSignMsg{})
+	if err != nil {
+		fmt.Printf("passphrase error: %v", err)
```

Comment:
ditto

Reply:
fixed in next push

Reply:
https://github.com/cosmos/peggy/commit/102c1283899e9820cd4c2123eb5d2b94b75ad070